runtime/compiler/env/VMJ9.cpp

```diff
@@ -6498,9 +6498,13 @@ TR_J9VMBase::isEnumClass(TR_OpaqueClassBlock * clazzPointer, TR_ResolvedMethod *
 bool
 TR_J9VMBase::isAbstractClass(TR_OpaqueClassBlock * clazzPointer)
    {
-   if (isInterfaceClass(clazzPointer))
-      return false;
-   return (TR::Compiler->cls.romClassOf(clazzPointer)->modifiers & J9AccAbstract) ? true : false;
+   return (TR::Compiler->cls.romClassOf(clazzPointer)->modifiers & (!J9AccInterface & J9AccAbstract)) ? true : false;
```

Comment:
`!` is the logical not. What you want here is to isolate the two bits you are interested in, and then test the have the values you want:
`(modifiers & (J9AccInterface | J9AccAbstract)) == J9AccAbstract`